<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>s21_deposit</class>
 <widget class="QWidget" name="s21_deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>880</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #00A876;
</string>
  </property>
  <widget class="QDoubleSpinBox" name="spin_amount">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
		border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000099.989999999990687</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_term">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
		border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>20</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}

</string>
   </property>
   <property name="text">
    <string>Deposit amount</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spin_rate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
		border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000099.989999999990687</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>90</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}

</string>
   </property>
   <property name="text">
    <string>Interest rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>160</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}

</string>
   </property>
   <property name="text">
    <string>Interest time</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spin_tax_rate">
   <property name="geometry">
    <rect>
     <x>668</x>
     <y>60</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 20px;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000099.989999999990687</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}

</string>
   </property>
   <property name="text">
    <string>Tax rate</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_begin">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>150</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 20px;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}

</string>
   </property>
   <property name="text">
    <string>Months begin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}

</string>
   </property>
   <property name="text">
    <string>Frequency of payments</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_capital">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>260</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px;</string>
   </property>
   <property name="text">
    <string>Капитализация</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_payments_give">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font-size: 25px;
background-color: white;
border-top-left-radius: 20;
border-bottom-left-radius: 20;</string>
   </property>
   <item>
    <property name="text">
     <string>Ежедневно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>В конце срока</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="button_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-size: 30px;
	color: white;
	background-color: #F2A918;
	border-style: outset;
	border-width: 2px;
	border-color: beige;
	border-radius: 30px;
}</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>851</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	font-size: 25px;
}</string>
   </property>
   <property name="title">
    <string>Пополнения</string>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size: 25px;
border-radius: 20px;
}

</string>
    </property>
    <property name="text">
     <string> Summary add</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_summary_add">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox{
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
		border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
}</string>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000099.989999999990687</double>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_add">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
font-size: 25px;
background-color: white;
border-top-left-radius: 20;
border-bottom-left-radius: 20;</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Раз в месяц</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Раз в квартал</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Раз в полгода</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Раз в год</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string>RUB</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>851</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	font-size: 25px;
}</string>
   </property>
   <property name="title">
    <string>Частичные снятия</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size: 25px;
border-radius: 20px;
}

</string>
    </property>
    <property name="text">
     <string> Summary back</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_summary_sub">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox{
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
		border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
}</string>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000099.989999999990687</double>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_sub">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
font-size: 25px;
background-color: white;
border-top-left-radius: 20;
border-bottom-left-radius: 20;</string>
    </property>
    <item>
     <property name="text">
      <string>Раз в месяц</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Раз в квартал</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Раз в полгода</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Раз в год</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string>RUB</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_calculate">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>570</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-size: 30px;
	color: white;
	background-color: #F2A918;
	border-style: outset;
	border-width: 2px;
	border-color: beige;
	border-radius: 30px;
}</string>
   </property>
   <property name="text">
    <string>CALCULATE</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spin_tax_end">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>670</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
		border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000099.989999999990687</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}

</string>
   </property>
   <property name="text">
    <string>Начисленные проценты</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spin_symmary_tax_end">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>740</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
		border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000099.989999999990687</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}

</string>
   </property>
   <property name="text">
    <string>Сумма налога</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spin_sum_end">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>810</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
		border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000099.989999999990687</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>800</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}

</string>
   </property>
   <property name="text">
    <string>Сумма на вкладе к концу срока</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>560</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(&quot;/Users/lymondgl/Desktop/s21_SmartCalc-v2.0/src/images/sber.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>740</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>RUB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>670</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>810</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>RUB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>Days</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>RUB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
